### PR TITLE
login front in react

### PR DESCRIPTION
**Pull request by vedantjain0087**
Shivam kindly check the code for front login in react

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "start": "npm run build",
+    "build": "webpack -d && webpack-dev-server --content-base src/ --inline --hot --port 1234 --history-api-fallback"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "autoprefixer": "^8.6.2",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.4",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (shivam00) [inline package.json:17]: do you know about every dependencies you included in this proj kindly open each open and get info about them

File: src/app/assets/css/demo.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+.demo-button button {
+  margin-bottom: 5px; }
+
+.demo-icons li {
+  margin-bottom: 50px;
+  text-align: center; }
+
+.demo-icons .lnr,
+.demo-icons .fa {
+  font-size: 28px; }
```
Comment (shivam00) [inline src/app/assets/css/demo.css:11]: mark it in % for responsiveness else use a media query to distinguish  between screen and use them

File: src/app/assets/scripts/klorofil-common.js [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+		});
+
+		$('#toastr-callback3').on('click', function() {
+			$message = $(this).data('message');
+
+			toastr.options = {
+				"timeOut": "10000",
+				"closeButton": true,
+				"onCloseClick": function() { alert('onCloseClick callback'); }
```
Comment (shivam00) [inline src/app/assets/scripts/klorofil-common.js:200]: why using Jquery when you are using react ?
do we really need 2 lib ??
whats the need of jquery here??

**Review by shivam00 — CHANGES_REQUESTED** [review]

File: src/app/assets/css/demo.css (hunk 2 above)
Comment (vedantjain0087) [reply src/app/assets/css/demo.css:11]: It was Already defined in the theme ... yes it is responsive

File: src/app/assets/scripts/klorofil-common.js (hunk 3 above)
Comment (vedantjain0087) [reply src/app/assets/scripts/klorofil-common.js:200]: ok sir will remove it

File: package.json (hunk 1 above)
Comment (vedantjain0087) [reply package.json:17]: ok sir

**Pull request merged**